### PR TITLE
TA-458 Test discrimination claims manual submission and bulkload pricing

### PR DESCRIPTION
**Pull request by your**
## What does this pull request do?

What the title says.

## Why make these changes?

We need to test the new screen to manually add Discrimination outcomes to CWA. A successful save would also validate that we have added the right code combinations (even if we don't test them _all_) and the pricing is correct. We're also testing pricing an a number of cases that should cover the requirements.

## Technical

One of the main focus of this PR is to move the existing `OutcomePage` content to a more adeguate `SitePrism`'s Page Object with all possible elements defined at runtime, and used when needed based on the test via a `values` hash. The proposed solution might need to be revised now or soon if we start testing more screens with way too many different field mappings.

## Checklist

- [x] Provided link to a JIRA ticket: https://dsdmoj.atlassian.net/browse/TA-458
- [x] Related: https://github.com/ministryofjustice/laa-cwa/pull/53

Please note:

* ~~this PR does not test Pricing, which could be done in a separate card instead in order to still meet the Spring goal (setup screen, without pricing)~~ it's testing both things now.
* ~~it currently fails in SIT10 because of a Pricing error which has been fixed by @GlenHarper in DEV20 but not applied to SIT10 as yet. Codewise ready for review.~~
* we are temporarily skipping the step where we validate the escape fee flag as it's currently not working and will be addressed with another card TA-472.

### CODE REVIEW

File: features/discrimination_outcome_pricing.feature [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+  Scenario Outline: Discrimination outcome pricing
+    When user is looking at outcome "<UFN>"
+    Then user should see the outcome priced at "£ <Price>"
+    And the Escape Fee flag is "<Escape Fee>"
+
+    Examples:
+      | UFN        | Profit ex. VAT | Counsel ex. VAT | Disbursements ex. VAT | Disbursements VAT | Price  | Escape Fee |
+      | 010919/001 | 699.99         | 0.00            | 0.00                  | 0.00              | 699.99 | N          |
+      | 010919/002 | 699.99         | 0.00            | 100.00                | 20.00             | 719.99 | N          |
```
Comment (solomonahmed) [inline features/discrimination_outcome_pricing.feature]: Profit ex. VAT + Counsel ex. VAT < 700 then Price should include Disbursements ex. VAT + Disbursements VAT.

Also, the VAT_INDICATOR if set to Y in the outcome record will mean VAT @ 20% is applied to the Profit ex. VAT and included in the Price.

**Comment by your** [comment]
```
54 scenarios (54 passed)
240 steps (240 passed)
9m44.343s
```

File: features/step_definitions/cwa_outcome_steps.rb [hunk 2]
```diff
@@ -90,27 +95,73 @@
...
+  page.add_outcome(values)
+end
+
+When('user adds a valid outcome for Discrimination') do
+  submission_list_page = SubmissionListPage.new
+  submission_list_page.add_outcome_button.click
+
+  page = AddOutcomePage.new
+  values = {
+    matter_type: 'QPRO:QAGE',
```
Comment (solomonahmed) [inline features/step_definitions/cwa_outcome_steps.rb]: Be good in the future to re-factor out of this step the hard-coded outcome data as discussed.  Separating out data from logic.  One for the future.

File: features/add_outcome_manually.feature [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Feature: Add outcome manually
+
+  @delete_outcome_after
+  Scenario: Add an outcome for Discrimination
+    Given user is on their sumission details page
```
Comment (solomonahmed) [inline features/add_outcome_manually.feature]: Shouldn't  'sumission' be submission?

File: features/step_definitions/cwa_outcome_steps.rb (hunk 2 above)
Comment (your) [reply features/step_definitions/cwa_outcome_steps.rb]: 100% agreed. These can be moved to a fixture file in yaml format for instance. Can do in the next iteration 👍.

File: features/add_outcome_manually.feature (hunk 3 above)
Comment (your) [reply features/add_outcome_manually.feature]: Good point, fixing that.

Comment (your) [reply features/add_outcome_manually.feature]: Addressed with f656a8f.

**Pull request merged**